zipkin-server/README.md

```diff
@@ -17,6 +17,10 @@ The following endpoints are defined for Zipkin:
 
 There are more [built-in endpoints](https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-endpoints.html) provided by Spring Boot, such as `/metrics`. To comprehensively list endpoints, `GET /mappings`.
 
+### CORS (Cross-origin Resource Sharing)
+
+By default, all end points under `/api/v1` are configured to **allow** cross-origin requests. This can be changed by setting the `zipkin.query.allowed-origins` property.
```

Comment:
last nits.. we are using "endpoint" not "end point". Then put an example, so people know what the value should be. something like...

For example, to only allow requests from mycompany.com, you can set the environment variable `ZIPKIN_QUERY_ALLOWED_ORIGINS=mycompany.com`.

(pretty sure that's the syntax when set invoked as an env variable)